apps/rewards/contracts/Rewards.sol

```diff
@@ -62,8 +62,8 @@ contract Rewards is AragonApp {
     Vault public vault;
 
     /// Events
-    event RewardAdded(uint256 rewardId); /// Emitted when a new reward is created
-    event RewardClaimed(uint256 rewardId); /// Emitted when a reward is claimed
+    event RewardAdded(uint256 rewardId, address adder);
```

Comment:
I don't think keeping the comments would hurt, but on the other hand the events names seem quite intuitive, so I am "ok" with the change.

Reply:
Yeah, a bit redundant and unnecessary. But if they're part of some convention or standard practice, it might make sense to leave them.